### ISSUE TITLE
Multi-page signup flow

### ISSUE THREAD

**Issue by markulrich** [rahafoundation/raha-web-app#91]
Talked with Tina about sample multi-page invite flow featuring simple idea per page with tap/arrow to continue, eg

1. Tina invited you to Raha.io Network, an identity network supporting universal basic income.
1. We believe every life has value, but financialization and automation is creating unsustainable and unethical inequality.
1. Members of the Raha identity network receive a basic income of Raha coin. In exchange, when you are inactive for a year or more then your coin are donated back to the network.
1. Raha is free to join with 0 transaction fees. Talk to Tina to create your video identity!
1. Sign in and agreements
1. Video upload
1. Your new profile (which says pending at the top)

**Comment by markulrich** [comment]
During step #5 we should have checkboxes
- [ ] I understand that after 1 year of no transactions all of my Raha will be irrevocably donated to fund basic income, with 80% going directly to members and 20% to the Raha Foundation.
- [ ] I agree to the community code of conduct, terms of service, and privacy policy.
- [ ] This is my real identity and only time I have joined Raha. I understand that creating duplicate or fake accounts may result in me and people I have invited losing access to our accounts.
